staging/src/k8s.io/cloud-provider/config/v1alpha1/defaults.go

```diff
@@ -69,3 +70,78 @@ func SetDefaults_KubeCloudSharedConfiguration(obj *KubeCloudSharedConfiguration)
 		obj.RouteReconciliationPeriod = metav1.Duration{Duration: 10 * time.Second}
 	}
 }
+
+// SetDefaults_ValidatingWebhook sets defaults for webhook validating. This function
+// is duplicated from "k8s.io/kubernetes/pkg/apis/admissionregistration/v1/defaults.go"
+// in order for in-tree cloud providers to not depend on internal packages.
+func SetDefaults_ValidatingWebhook(obj *admissionregistrationv1.ValidatingWebhook) {
```

Comment:
I don't think any defaulting is required here... the defaulting gets applied when creating against the server